### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**

### CODE REVIEW

**Comment by rabbah** [comment]
Thank you for correction our bad spelling and typos :)

File: tests/src/common/WskCli.java [hunk 1]
```diff
@@ -637,7 +637,7 @@ public RunResult invoke(int expectedCode, String name, Map<String, String> param
      *
      * @param name
      *            the name of the action.
-     * @param expecte
+     * @param expected
      *            dthe expected activation id
```
Comment (rabbah) [inline tests/src/common/WskCli.java]: note `d` on `dthe`.

File: tools/cli/wskadmin [hunk 2]
```diff
@@ -211,6 +211,7 @@ def getUserCmd(args, props):
         print 'Failed to get subject (%s)' % res.read().strip()
         return 1
 
+# This should be getSubjectFromDb
 def getSubjecFromDb(args, props):
```
Comment (rabbah) [inline tools/cli/wskadmin]: Thanks - will fix.

File: tools/go-cli/go-whisk/whisk/shared.go [hunk 3]
```diff
@@ -20,7 +20,7 @@ import "encoding/json"
 
 type KeyValue struct {
     Key   string        `json:"key,omitempty"`
-    Value interface{}   `json:"value"`     // Whisk permits empty values, do don't omitemtpy
+    Value interface{}   `json:"value"`     // Whisk permits empty values, do don't omitempty
```
Comment (rabbah) [inline tools/go-cli/go-whisk/whisk/shared.go]: @dubeejw @mdeuser is this supposed to be `don't omit empty`? the commit doesn't make any sense.

File: tools/cli/wskadmin (hunk 2 above)
Comment (jsoref) [inline tools/cli/wskadmin]: It's hard to know what's a public api, so I chose not to "just fix it". Also, it needed to be called out as more significant than the rest.

File: tests/src/common/WskCli.java (hunk 1 above)
Comment (jsoref) [inline tests/src/common/WskCli.java]: Yes. It was in the list of words for me to look at.

I was considering 2324 words and apparently didn't check that one.

Basically, my tool calculates things that are tokens and excludes things it knows are words, and then leaves me to decide which are probably typos vs. which are probably intentional identifiers (ansible)/actual linenoise (AQAABAAAAABQ).

File: tools/go-cli/go-whisk/whisk/shared.go (hunk 3 above)
Comment (jsoref) [inline tools/go-cli/go-whisk/whisk/shared.go]: probably "so don't omit empty"

**Comment by rabbah** [comment]
Made few more tweaks here https://github.com/openwhisk/openwhisk/commit/af1f1cc4ce086aee0c8424838480c8ef67082738. If it's alright with you I will squash all the commits (including mine) into one and commit.

**Comment by jsoref** [comment]
Go ahead

**Comment by rabbah** [comment]
Rebased, squashed and merged as https://github.com/openwhisk/openwhisk/commit/6386717735f0f24e8eba90db4395292e5ea0c2ee.

Thanks for the fixes!!

**Comment by csantanapr** [comment]
Hi @jsloyer I'm glad to hear back from you, I remember you from Apache Cordova project.
It looks like you had fun fixing our spelling errors 😄 
I'm curious if you have any plans on using OpenWhisk on any of your current projects. 
It would be valuable to hear your feedback and point of view on the project the good, bad and the ugly. 
You know how to reach me if you want to have a short hangout, as always don't be shy to reach out for anything here on Github or our Slack  👍

**Comment by csantanapr** [comment]
ping @jsoref

**Comment by jsoref** [comment]
It's a possibility. The reason I touched this project was because a company mentioned it in their focus.

**Pull request closed without merging**